sdk/storage/storage-blob/package.json

```diff
@@ -78,6 +78,15 @@
     "clean": "rimraf dist dist-esm dist-test typings temp browser/*.js* browser/*.zip statistics.html coverage coverage-browser .nyc_output *.tgz *.log test*.xml TEST*.xml",
     "extract-api": "tsc -p . && api-extractor run --local"
   },
+  "files": [
+    "BreakingChanges.md",
+    "browser/*.min.js*",
```

Comment:
In the browser folder, there's a txt file which is supposed to be included.

Reply:
Good catch, we also need `browser/ThirdPartyNotices.txt`.

Reply:
added :)